easybuild/easyconfigs/c/Clp/Clp-1.17.3-foss-2018b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Clp'
+version = '1.17.3'
+
+homepage = "https://github.com/coin-or/Clp"
+description = """Clp (Coin-or linear programming) is an open-source linear programming solver.
+ It is primarily meant to be used as a callable library, but a basic, 
+ stand-alone executable version is also available."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://www.coin-or.org/download/source/%(name)s/']
+sources = [SOURCE_TGZ]
+checksums = ['a13bf54291ad503cf76f5f93f2643d2add4faa5d0e60ff2db902ef715c094573']
+
+builddependencies = [
+    ('Doxygen', '1.8.14'),
+]
+
+dependencies = [
+    ('OpenBLAS', '0.3.1'),
+    ('ScaLAPACK', '2.0.2', '-OpenBLAS-0.3.1'),
```

Comment:
@tstrempel those 2 deps are loaded by default by the `foss` toolchain, so you can remove them.

Reply:
@smoors I didn't know, that the math libraries are implicitly added to the module file. I'll update my pull request.